v2/pkg/output/output.go

```diff
@@ -224,3 +224,38 @@ func (w *StandardWriter) Close() {
 		w.errorFile.Close()
 	}
 }
+
+// MockOutputWriter is a mocked output writer.
+type MockOutputWriter struct {
```

Comment:
Test code should not be in the same place with production code. It should be extracted somewhere (preferably under the `testutils` package, if possible)